src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -5281,6 +5281,36 @@ public void NamespaceReorder2()
             edits.VerifyRudeDiagnostics();
         }
 
+        [Fact]
+        public void Namespace_FileScoped_Insert()
+        {
+            var src1 = @"";
+            var src2 = @"namespace C;";
+
+            var edits = GetTopEdits(src1, src2);
+
+            edits.VerifyEdits(
+                "Insert [namespace C;]@0");
+
+            edits.VerifyRudeDiagnostics(
+                Diagnostic(RudeEditKind.Insert, "namespace C", FeaturesResources.namespace_));
+        }
+
+        [Fact]
+        public void Namespace_FileScoped_Delete()
+        {
+            var src1 = @"namespace C;";
+            var src2 = @"";
+
+            var edits = GetTopEdits(src1, src2);
+
+            edits.VerifyEdits(
+                "Delete [namespace C;]@0");
+
+            edits.VerifyRudeDiagnostics(
+                Diagnostic(RudeEditKind.Delete, null, FeaturesResources.namespace_));
+        }
+
```

Comment:
Since file scoped namespaces can't appear with top level code, can't be nested, and can't have multiple in a file, these two tests count as full coverage 😛